website/src/css/pages/_jobs.scss

```diff
@@ -35,7 +35,7 @@
         display: none;
 
         @media only screen and (max-width: 768px) {
-            display: block;
+            display: inline-block;
```

Comment:
I hesitate to suggest this (considering how all over the place the margins and padding are), but would we want margin on the mobile image, to at least sort of match the text blocks and other items on the page?

Reply:
Yeah, there's a lot of inconsistencies and style implementations that we need to clean up.

This should probably be a question for design but I do like the full-bleed images. We could remove the max-width constraint for the mobile image so from mobile up to the 768px breakpoint, the mobile image is still full-bleed. What do you think @stephzabala? Do you like the full-bleed images or do you want them floating within certain width constraints as they are now? Check [the jobs page](https://deploy-preview-5119--sourcegraph.netlify.app/jobs/) for responsiveness and [this loom](https://www.loom.com/share/88ff27b77a0d4e63be716c3765880c38).

Reply:
@stephzabala Great, sounds good! Yes, it's a lot more fluid from breakpoint to breakpoint and the images are full-bleed to the horizontal edges of the viewport 🤓